Submitted By:            Igor Živković <contact@igor-zivkovic.from.hr>
Date:                    2013-10-18
Initial Package Version: 1.5.2
Upstream Status:         Unknown
Origin:                  Debian
Description:             Fixes building.

diff -Naur nitrogen-1.5.2.orig/configure.ac nitrogen-1.5.2/configure.ac
--- nitrogen-1.5.2.orig/configure.ac	2011-04-20 03:52:10.000000000 +0200
+++ nitrogen-1.5.2/configure.ac	2013-10-18 10:16:36.334515994 +0200
@@ -34,9 +34,9 @@
 PKG_CHECK_MODULES(GTK2,[gtk+-2.0 >= $GTK2_VERSION])
 PKG_CHECK_MODULES(GTKMM2, [gtkmm-2.4 >= $GTKMM2_VERSION])
 PKG_CHECK_MODULES(GTHREAD2, [gthread-2.0 >= $GTHREAD2_VERSION])
+PKG_CHECK_MODULES(X11, x11)
 
-
-NITROGEN_LIBS="$GLIB2_LIBS $GTK2_LIBS $GTKMM2_LIBS $GTHREAD2_LIBS"
+NITROGEN_LIBS="$GLIB2_LIBS $GTK2_LIBS $GTKMM2_LIBS $GTHREAD2_LIBS $X11_LIBS"
 NITROGEN_CFLAGS="$GLIB2_CFLAGS $GTK2_CFLAGS $GTKMM2_CFLAGS $GTHREAD2_CFLAGS"
 
 dnl do some autotools picen to get inotify
